react-form-controls/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormEvent } from 'react';
+import { useState } from 'react';
+
+export function RegistrationFormControlled() {
+  const [user, setUser] = useState('');
+  const [pass, setPass] = useState('');
+
+  function handleSubmit(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    console.log(`username: ${user}, password: ${pass}`);
+  }
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Username
+        <input
+          name="username"
+          onChange={(e) => {
+            setUser(e.target.value);
```

Comment:
On these, the `{}` aren't required. It makes the code a bit cleaner to leave them off.